src/components/Loader/test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+
+import Loader from '.';
+import { renderWithTheme } from 'utils/tests/helpers';
+
+describe('<Loader />', () => {
+  it('should have a div with a z-index in overlay level (30)', () => {
+    const { container } = renderWithTheme(<Loader />);
+
+    expect(container.firstChild).toHaveStyle('z-index: 30');
+  });
+
+  it('should have a div with the whole screen size and to be a fixed element', () => {
+    const { container } = renderWithTheme(<Loader />);
+
+    expect(container.firstChild).toHaveStyle('height: 100%');
```

Comment:
**Suggestion**:

Instead of
```
expect(container.firstChild).toHaveStyle('height: 100%');
expect(container.firstChild).toHaveStyle('width: 100%');
expect(container.firstChild).toHaveStyle('position: fixed');
```

You can use
```
expect(container.firstChild).toHaveStyle(`
  height: 100%';
  position: fixed;
  width: 100%;
`);
```

Or
```
expect(container.firstChild).toHaveStyle({
  height: 100%',
  position: fixed,
  width: 100%
});
```

https://github.com/testing-library/jest-dom#tohavestyle